packages/protocol/lib/test-utils.ts

```diff
@@ -208,33 +210,38 @@ export async function waitForPortOpen(host: string, port: number, seconds: numbe
 }
 
 export const assertProxiesSet = async (getContract: any) => {
```

Comment:
Since we are touching these methods it would be nice to have signature strongly typed